apps/condo/domains/common/utils/makeid.utils.js

```diff
@@ -0,0 +1,13 @@
+function makeId (length) {
+    let result = ''
+    const characters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789'
+    const charactersLength = characters.length
+    for (let i = 0; i < length; i++) {
+        result += characters.charAt(Math.floor(Math.random() * charactersLength))
```

Comment:
An util `sample` from Lodash can be used here.

Reply:
Done